fsh/ig-data/input/pages/index.md

```diff
@@ -229,6 +229,8 @@ patient selection widget.  For full details, see <a href="scopes-and-launch-cont
 *	launch/patient - to indicate that the app needs to know a patient ID
 *	launch/encounter - to indicate the app needs an encounter
 
+### Considerations for native apps
```

Comment:
This is written as a requirement for apps; would support be "SHALL" for servers?